composer.json

```diff
@@ -28,7 +37,11 @@
     "prefer-stable": true,
     "scripts": {
         "post-install-cmd": "sh ./scripts/composer/post-install.sh",
-        "docker-swagger-generate": "docker-compose run swagger generate -i http://service:3000/explorer/swagger.json -l php -o /var/usr/client -c /var/usr/swagger/config.json"
+        "docker-swagger-generate": "docker-compose run swagger generate -i http://service:3000/explorer/swagger.json -l php -o /var/usr/client -c /var/usr/swagger/config.json",
```

Comment:
Should the swagger run as part of the pipeline?
Alternative: 
http://zircote.com/swagger-php/using_swagger.html#via-command-line
```
php swagger.phar /projects/my_project  -o /var/html/swagger-docs
```